llabake/Hello-Books
# Build common component 153734915

#### What does this PR do?
This PR add common components to the app
#### Description of Task to be completed?
App should be able to load the page header and footer
#### How should this be manually tested?
clone the repo
cd into the project directory
Run npm install to install all the dependency
Run npm run react:dev
visit the port where the webpack-dev-server started, the header and footer components should be loaded
#### What are the relevant pivotal tracker stories?
153734915
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/34117669-ac610eae-e41c-11e7-9fec-e444381a38ea.png)
